src/enable_variable_ops/ngraph_catalog.cc

```diff
@@ -0,0 +1,107 @@
+/*******************************************************************************
+ * Copyright 2019-2020 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+#include "tensorflow/core/lib/core/errors.h"
+
+#include "ngraph/ngraph.hpp"
+#include "ngraph/runtime/backend_manager.hpp"
+#include "ngraph_catalog.h"
+#include "ngraph_log.h"
+
+using namespace std;
+namespace ng = ngraph;
+
+namespace tensorflow {
+
+namespace ngraph_bridge {
+
+unordered_map<string, string> NGraphCatalog::input_variable_map_;
+map<string, shared_ptr<ng::runtime::Tensor>> NGraphCatalog::output_tensor_map_;
+unordered_map<string, unordered_set<int>>
+    NGraphCatalog::ng_encap_output_copy_map_;
```

Comment:
Lines 30-33 do not do anything right? They are already declared in the `.h` file